docs/atomic-run.1.md

```diff
@@ -49,6 +49,9 @@ These defaults are suggested values for your container images.
 **SUDO_GID**
   The `SUDO_GID` environment variable.  This is useful with the docker `-u` option for user space tools.  If the environment variable is not available, the default GID of the value for `SUDO_UID` is used.  If this value is not available, the value of `/proc/self/loginuid` is used.
 
+**RUN_OPTS**
```

Comment:
I am just thinking that once we have it for one option people will want it for others.
Lets keep it as RUN_OPTS_FILE and ignore the others for now, and we will create them if needed.